### DISCUSSION TITLE
Displaying .dds in Image

### DISCUSSION THREAD

**Discussion by iMrShadow** [AvaloniaUI/Avalonia#14775] [Q&A] [answered]
Hello! 
I want to be able to display .dds file, but the file format is not supported by default. Are there any other ways to display it? I can't find any sources about it.  

```csharp
 var bitmap = new Bitmap(fileDirectory.FilePath);

 ui_imagepreview_image.Source = bitmap;
```

```xaml
<Image Name="ui_imagepreview_image" Stretch="Uniform"/>
```

**Comment by maxkatz6** [comment]
I don't think Skia supports dds image formats. You can try to decode your images with this library https://github.com/nickbabcock/Pfim and then pass to Bitmap as a pointer or WriteableBitmap.

**Reply by maxkatz6** [reply, marked as answer]
Something similar, but adapted for Avalonia https://github.com/nickbabcock/Pfim/blob/master/src/Pfim.Skia/Program.cs#L21
